### PR TITLE
added user_id to posts. User name can be retrieved when showing posts

### PR DESCRIPTION
**Pull request by ErlantzR**

### CODE REVIEW

File: controllers/posts.js [hunk 1]
```diff
@@ -2,11 +2,14 @@ const Post = require("../models/post");
...
 const PostsController = {
   Index: (req, res) => {
-    Post.find((err, posts) => {
+
+    Post.find().populate('user_id').exec((err, posts) => {
       if (err) {
         throw err;
       }
 
+      console.log(posts);
```
Comment (lukestorey95) [inline controllers/posts.js:11]: Remove console log

File: spec/models/post.spec.js [hunk 2]
```diff
@@ -15,6 +15,11 @@ describe("Post model", () => {
     expect(post.message).toEqual("some message");
   });
 
+  it.skip("has a user id", () => {
```
Comment (lukestorey95) [inline spec/models/post.spec.js:18]: Why is this skipped?

**Review by lukestorey95 — CHANGES_REQUESTED** [review]

File: controllers/posts.js (hunk 1 above)
Comment (ErlantzR) [reply controllers/posts.js]: Done

File: spec/models/post.spec.js (hunk 2 above)
Comment (ErlantzR) [reply spec/models/post.spec.js]: Apparently the last modification we made to this test was not committed. At that point, that test was not working. I have pushed a new commit with that test modified and passing

**Review by lukestorey95 — CHANGES_REQUESTED** [review]

File: spec/models/post.spec.js [hunk 3]
```diff
@@ -15,6 +15,11 @@ describe("Post model", () => {
     expect(post.message).toEqual("some message");
   });
 
+  it("has a user id", () => {
+    var post = new Post({ message: "some message", user_id: "some user id"});
+    expect(post.user_id).not.toBeNull();
```
[the remark this answers is not in the document]
Comment (ErlantzR) [reply spec/models/post.spec.js:20]: I don't think you are wrong but, it's not that tested already on the integration test? This test it is just testing that the post is accepting something on that field. Would that be enough?

[the remark this answers is not in the document]
Comment (lukestorey95) [reply spec/models/post.spec.js:20]: In my mind the saving of the post is testing the schema being set up correctly. A change has been made to the schema so I think it makes sense to update the tests to reflect that change. Agree that the integration test technically covers it, but as we continue to update the post schema I would personally like to have tests for the updates.

**Pull request merged**